assets/app/view/game/bid.rb

```diff
@@ -29,6 +31,7 @@ def render
             process_action(Engine::Action::Bid.new(
               @entity,
               corporation: @corporation,
+              company: @company,
```

Comment:
Again, generalize here. Under-the-hood, you can use `corporation?` and `company?` to store the type for serialization purposes.

Reply:
Do we have a good name for the combination of corporation and company the same variable. Entity is already used for the player.

Reply:
Perhaps change entity to bidder and then use entity?